cekit/templates/template.jinja

```diff
@@ -23,30 +49,24 @@ USER root
 {% if packages and packages.install %}
 {% if packages.repositories_injected %}
 # Add custom repo files
-COPY \ {% for repo in packages.repositories_injected %} 
-    repos/{{ repo.filename }} \
-{% endfor %}
-    /etc/yum.repos.d/
+COPY {{ repo_create(pkg_manager, packages.repositories_injected) }}
 
 {% endif %}
 {% if packages.repositories %}
 {% for repo in packages.repositories %}
 {% if repo.present and repo.rpm %}
-RUN yum install -y {{ repo.rpm }} \
-    && yum clean all && rm -rf /var/cache/yum
+RUN {{ repo_install(pkg_manager, repo.rpm) }}
 
 {% endif %}
 {% endfor %}
 {% endif %}
 
 # Install required RPMs and ensure that the packages were installed
-RUN yum install -y {%- for package in packages.install %} {{ package }}{% endfor %} \
-    && yum clean all && rm -rf /var/cache/yum && \
-    rpm -q {% for package in packages.install %} {{ package }}{% endfor %}
```

Comment:
NACK at the moment because this line is not represented in the new macro. With `yum`, it will return a success status code if any package you asked for is installed: even if another did not. This line is necessary to ensure the build fails if a package fails to install. Please incorporate that into the macro.

Reply:
It is there https://github.com/cekit/cekit/blob/4a3e007e3aefbfd3c34b736ef0f01fc3c86b367a/cekit/templates/template.jinja#L22-L24